examples/hooks/components/layout/Tabs.css

```diff
@@ -0,0 +1,43 @@
+.Tabs-header {
+  display: flex;
+}
+
+.Tabs-title {
+  padding: 0 1rem;
+  flex: 1;
+  font-family: inherit;
+  font-size: .875rem;
+  color: #23263b;
+  background: linear-gradient(-180deg, #fff, #fcfcfd);
+  border: 1px solid #d6d6e7;
+  box-shadow: 0 1px 0 0 rgb(35 38 59 / 5%);
+  text-align: center;
+  line-height: 2rem;
+  cursor: pointer;
+}
+
+.Tabs-title:first-child {
+  border-radius: 3px 0 0 3px;
+}
+
+.Tabs-title:last-child {
+  border-radius: 0 3px 3px 0;
+}
+
+.Tabs-title:hover {
+  background: linear-gradient(-180deg, #fff, #f5f5fa);
+}
+
+.Tabs-title + .Tabs-title {
+  margin-left: -1px;
+}
+
+.Tabs-title--active {
+  font-weight: bold;
+  box-shadow: inset 0 1px 4px 0 rgb(119 122 175 / 40%), inset 0 1px 1px 0 rgb(119 122 175 / 40%), 0 1px 0 0 rgb(35 38 59 / 5%);
+  z-index: 1;
+}
+
+.Tabs-list {
+  margin-top: 1rem;
+}
```

Comment:
Great job on the tabs! 
I would make an extra effort and have the example follow the [satellite tabs UI](https://algolia-satellite.netlify.app/?path=/docs/navigation-tabs--basic).
I would either replicate the styles or use their tabs component. 

https://algolia-satellite.netlify.app/?path=/docs/navigation-tabs--basic

We usually don't put a lot of effort in the look of our demos, let's try to change that.

Reply:
Didn't know we had that. I'll look into it and it might be better to use their tabs component indeed. Thanks!

Reply:
I think we have gone back-and-forth too long about these tabs already, and what's used now fits the design as well, and is also accessible. We can swap it out for either the satellite or another third party later. Don't forget this is just an example!

Reply:
Okay, I took a few minutes to try it out and their CSS overrides some of our own styles, so there might be some more work involved. I'll merge it for now if that's okay.